app/models/contact.rb

```diff
@@ -23,7 +23,7 @@ class Contact < ApplicationRecord
   after_create :republish_organisation_to_publishing_api
   after_destroy :republish_organisation_to_publishing_api
 
-  # after_commit :republish_embassies_index_page_to_publishing_api, :republish_worldwide_office_to_publishing_api
+  # after_commit :republish_embassies_index_page_to_publishing_api
```

Comment:
Looks like this was and still is commented out - should it be?

Reply:
It was intentionally disabled in https://github.com/alphagov/whitehall/pull/9280 until the embassies index page issue is resolved.